dev/tasks/python-wheels/github.linux.yml

```diff
@@ -64,15 +64,15 @@ jobs:
         env:
           GITHUB_TOKEN: {{ '${{ secrets.GITHUB_TOKEN }}' }}
         run: |
-          if [ "{{ manylinux_version }}" = "2014" ] && [ "{{ arch }}" = "arm64" ]; then
+          if [ "{{ linux_wheel_version }}" = "2014" ] && [ "{{ arch }}" = "arm64" ]; then
             # We can't use NuGet on manylinux2014_aarch64 because Mono is old.
             :
-          else
+          elif [ "{{ linux_wheel_name }}" = "many" ]; then
```

Comment:
Can we install Mono in `python-wheel-musllinux.docker` instead?

Reply:
What you mean? On musl I don't think we need to do anything of this.

Reply:
This cache mechanism improves CI speed very much. So we want to use this as much as possible.

Reply:
I checked the `install_vcpkg.sh` script and it tries to use mono, that's probably why you asked me to install mono.
It's possible to install by building the sources but it takes quite a while. The other alternative is to install it via apk but we need to add the testing repository and the mono package is not in the main repository. Maybe we can install it via apk to improve CI speed. Does this sound right? Do you see a problem with adding the testing repository?

Reply:
> I checked the `install_vcpkg.sh` script and it tries to use mono, that's probably why you asked me to install mono.

Right.

> The other alternative is to install it via apk but we need to add the testing repository and the mono package is not in the main repository.

Oh...

Can we use `apk add mono --repository=https://dl-cdn.alpinelinux.org/alpine/edge/testing`?

Reply:
Oh yes that's much better

Reply:
Can we revert this change so that we use cache?